### PR TITLE
Upgrade to AutoMapper 13.0

### LINKED ISSUE #181: AutoMapper 13 support
The current constraint is: AutoMapper.Collection 9.0.0 -> AutoMapper (>= 12.0.0 && < 13.0.0).

### PR DESCRIPTION
**Pull request by bitschubse**
Fixes #181

### CODE REVIEW

**Comment by bitschubse** [comment]
@dotnet-policy-service agree

File: Directory.Build.props [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <EFVersion>6.1.3</EFVersion>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' or '$(TargetFramework)' == 'netcoreapp3.0' ">
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' or '$(TargetFramework)' == 'netcoreapp3.0' or '$(TargetFramework)' == 'net6.0' ">
```
Comment (lbargaoanu) [inline Directory.Build.props:15]: I think EFVersion can be removed. So two here and where it's used, just put the version.

Comment (bitschubse) [reply Directory.Build.props]: I've made the changes as suggested.

**Comment by mrsquish** [comment]
Thanks guys, much appreciated.

**Pull request merged**